tests/de/_common.yaml

```
@@ -0,0 +1,23 @@
+language: de
+areas:
+- name: Küche
+  id: kitchen
+- name: Wohnstube
```

```suggestion
- name: Wohnzimmer
```

Doesn't make a difference here. IMO this is the more common word.

---

it depends, in which region you are living 🙈 
But yes you're right, "Wohnzimmer" is the common

---

I'd agree with that. Wohnzimmer is more common.